src/components/venue-list/venue-list-component.scss

```diff
@@ -1,8 +1,15 @@
 .venue-list {
   padding: $global-padding;
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  min-height: 0;
```

Comment:
That's weird that you need this min-height, but played with it and its definitely necessary.  🤷‍♂

Reply:
Yeah, I thought the same thing. So strange.